conda/recipes/librmm/build.sh

```diff
@@ -1,3 +1,3 @@
 # Copyright (c) 2018-2022, NVIDIA CORPORATION.
 
-./build.sh -n -v clean librmm tests benchmarks --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
+./build.sh -n -v -g clean librmm tests benchmarks --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
```

Comment:
```suggestion
./build.sh -n -v clean librmm tests benchmarks --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
```

We should also revert this debug build flag as well